packages/shared/src/components/fields/form/FormWrapper.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+import { Button, ButtonVariant } from '../../buttons/Button';
+
+interface Copy {
+  left?: string;
+  right?: string;
+}
+
+interface FormWrapperProps {
+  children: ReactNode;
+  className?: string;
+  form: string;
+  copy?: Copy;
+  onLeftClick?(): void;
+  onRightClick?(): void;
+}
+
+export function FormWrapper({
```

Comment:
The idea is that this will be a seperate render for mobile, so we'll dynamic wrap or something?